docs/programming-languages/javascript-and-node-js.md

```diff
@@ -174,3 +172,40 @@ for details on pre-installed browsers and testing tools on Semaphore.
 [macos-javascript]: https://docs.semaphoreci.com/ci-cd-environment/macos-mojave-xcode-11-image/#javascript-via-node-js
 [docker-env]: https://docs.semaphoreci.com/ci-cd-environment/custom-ci-cd-environment-with-docker/
 [node-docker-image]: https://hub.docker.com/r/semaphoreci/node
+
+## Custom config.js
+
+You can create a custom config.js using [secrets](/guided-tour/environment-variables-and-secrets/#managing-sensitive-data-with-secrets) as files and environment variables.
```

Comment:
Please use prescriptive language in the docs, for guiding the reader.

Change `You can`, `You might` constructs into `Create a `, `Add a`.